llvm/include/llvm/DebugInfo/LogicalView/Core/LVScope.h

```diff
@@ -826,6 +828,23 @@ class LVScopeTemplatePack final : public LVScope {
   void printExtra(raw_ostream &OS, bool Full = true) const override;
 };
 
+// Class to represent a DWARF Module.
```

Comment:
Sorry, but we missed this one. It should be between `LVScopeFunctionType` and `LVScopeNamespace` to preserve the alphabetical order.

Reply:
Done :+1:.